icon-registry/icons/cancelled-outline_x16.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If you omit the width and height, the icons look way better in Mac Finder

```suggestion
<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
<img width="901" alt="Screen Shot 2022-07-22 at 10 53 00 AM" src="https://user-images.githubusercontent.com/5592465/180477066-5cb8cd7d-1fc2-475a-aef2-7bc48026f897.png">